packages/crystal/build.sh

```diff
@@ -0,0 +1,33 @@
+TERMUX_PKG_HOMEPAGE=https://crystal-lang.org/
+TERMUX_PKG_DESCRIPTION="A language for humans and computers"
+TERMUX_PKG_LICENSE="Apache-2.0"
+TERMUX_PKG_MAINTAINER="@leapofazzam123"
+TERMUX_PKG_VERSION=1.2.2
+TERMUX_PKG_SRCURL=https://github.com/crystal-lang/crystal/archive/refs/tags/$TERMUX_PKG_VERSION.tar.gz
+TERMUX_PKG_SHA256=6d963a71ef5f6c73faa272a0f81b50e9ddbf814b1ec07e557ce5c95f84d6077e
+TERMUX_PKG_AUTO_UPDATE=true
+TERMUX_PKG_DEPENDS="llvm, libllvm, libgc"
+TERMUX_PKG_BUILD_IN_SRC=true
+TERMUX_PKG_EXTRA_CONFIGURE_ARGS="
+release=1
+stats=1
+threads=$TERMUX_MAKE_PROCESSES
+LLVM_CONFIG=$TERMUX_PREFIX/bin/llvm-config
+CRYSTAL_CONFIG_VERSION=1.2.2
+CRYSTAL_CONFIG_PATH=lib:$TERMUX_PREFIX/lib/crystal
+"
+
+termux_step_make_install() {
+	install -Dm700 "$TERMUX_PKG_SRCDIR"/.build/crystal "$TERMUX_PREFIX"/bin/crystal
```

Comment:
This won't work. The source tarball download does not contain any binary files. You need to grab a distribution package from the release page for that.

Reply:
It might work in this case (if `crystal` is put into `.build/` by `make`), make is run automatically if a Makefile is found: [build-package.sh#L476](https://github.com/termux/termux-packages/blob/master/build-package.sh#L476), [termux_step_make.sh#L13](https://github.com/termux/termux-packages/blob/master/scripts/build/termux_step_make.sh#L13)